stack-overflow-app/src/Components/ViewQuestionPage.js

```diff
@@ -156,43 +156,77 @@ const ViewQuestionPage = () => {
         }
     };
 
+    const upVote= {
+
+        backgroundColor: "#04AA6D",
+    color: "white",
+    fontFamily: 'Source Sans Pro,sans-serif',
+    borderRadius: "5px",
+    border:'0',
+    Width: "60px",
+    marginLeft:'10px'
+    };
+    const downVote= {
+
+        backgroundColor: "#E55B13",
+    color: "white",
+    fontFamily: 'Source Sans Pro,sans-serif',
+    borderRadius: "5px",
+    border:'0',
+    Width: "60px",
+    marginLeft:'10px'
+    };
+    const BlueButton = {
+
+        backgroundColor: "#00868B",
+    color: "white",
+    fontFamily: 'Source Sans Pro,sans-serif',
+    borderRadius: "5px",
+    border:'0',
+    Width: "60px",
+    marginLeft:'10px'
+    };
+
+    const titleText = {fontFamily:'Teko',fontSize:'30px', paddingTop:'40px', color:'#008B8B', paddingLeft:'40px'};
+    const blueContainer = {backgroundColor:'#CCDCFD',marginLeft:'30px', marginRight:'50px', borderRadius:'5px'};
+
     return (
         <div>
-            <h1 style={{fontFamily:'Teko',fontSize:'30px', paddingTop:'40px', paddingLeft:'40px'}}>View Question</h1>
-            <div>
-                <h3 style={{fontSize:'15px', fontFamily:'sans-serif', paddingTop:'10px', paddingLeft:'60px'}}> Title: {state.question.title} </h3>
-                <h3 style={{fontSize:'15px', fontFamily:'sans-serif', paddingTop:'10px', paddingLeft:'60px'}}> Content: {state.question.content} </h3>
-                <p style={{fontSize:'15px', fontFamily:'sans-serif', paddingTop:'5px', paddingLeft:'60px'}}>Submitted by: {questionUsername}</p>
+            <h1 style={titleText}>View Question</h1>
+            <div style={blueContainer}>
+                <h3 style={{fontSize:'15px', fontFamily:'sans-serif', paddingTop:'10px', paddingLeft:'15px'}}> <b>Title: {state.question.title} </b> </h3>
```

Comment:
Remove "Title:" and "Content:" since they were placeholders used to identify them and are no longer necessary once styling is applied